programmers2/모의고사.js

```diff
@@ -0,0 +1,19 @@
+function solution(answers) {
+    const a = [1,2,3,4,5];
+    const b = [2,1,2,3,2,4,2,5];
+    const c = [3,3,1,1,2,2,4,4,5,5];
+    let cnt = [0, 0, 0];
```

Comment:
배열이므로 복수형 변수를 선언하는 것이 좋아요 :D
```suggestion
    const cnts = [0, 0, 0];
```